content/docs/for-buildpack-authors/concepts/component-buildpack.md

```diff
@@ -3,8 +3,22 @@ title="What is a component buildpack?"
 weight=99
 +++
 
+A **component buildpack** is a buildpack containing the `/bin/detect` and `/bin/build` executables and implements the [Buildpack Interface](https://github.com/buildpacks/spec/blob/main/buildpack.md#buildpack-interface).
+
 <!--more-->
 
-This page is a stub! The CNB project is applying to [Google Season of Docs](https://developers.google.com/season-of-docs/docs/timeline) to receive support for improving our documentation. Please check back soon.
+## Key Points
+
+During the `build` phase, typical component buildpacks might:
 
-If you are familiar with this content and would like to make a contribution, please feel free to open a PR :)
+* Read the Buildpack Plan in `<plan>` to determine what dependencies to provide
+* Provide the application with dependencies for launch in `<layers>/<layer>`
+* Reuse application dependencies from a previous image by appending `[types]` and `launch = true` to `<layers>/<layer>.toml`
+* Provide subsequent buildpacks with dependencies in `<layers>/<layer>`
```

Comment:
As the term `provides` has a technical definition in the Buildpacks world, do we think there might be a better term to use here?

Reply:
Thought I don't see an issue adding "provides", I changed the wording a bit. Hopefully I understood your point.